rcon/models.py

```diff
@@ -678,3 +678,9 @@ def valid_role(cls, values):
                 raise ValueError(f"Invalid Discord role {role_or_user}")
 
         return values
+
+
+class AdvancedConfigOptions(pydantic.BaseModel):
+    """ADVANCED_CRCON_SETTINGS in config.yml"""
+
+    thread_pool_size: pydantic.conint(ge=1, le=100)
```

Comment:
I'd catch and log the error that this is throwing in the init where you initialise that config.